app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -53,25 +53,26 @@ class MainActivity : ComponentActivity() {
     }
 }
 
+/*** Gets the current month and year and displays it above billing*/
 @Composable
 fun MainMenu() {
     //Getting current month for Main Menu
-    val cal: Calendar = Calendar.getInstance()
-    val monthDate = SimpleDateFormat("MMMM")
-    val monthName: String = monthDate.format(cal.time)
+    val calendar: Calendar = Calendar.getInstance()
+    val calendarFormatting = SimpleDateFormat("MMMM Y")
+    val formattedDate: String = calendarFormatting.format(calendar.time)
```

Comment:
It's probably a good idea to indicate how the date is being formatted, in the variable name, if there are multiple format options.